app/repositories/mandataris.js

```diff
@@ -0,0 +1,34 @@
+import { API, STATUS_CODE } from 'frontend-lmb/utils/constants';
+
+export default class MandatarisRepository {
+  async isActive(mandatarisId) {
+    const response = await fetch(
+      `${API.MANDATARIS_SERVICE}/mandatarissen/${mandatarisId}/isActive`
+    );
+    const jsonReponse = await response.json();
+
+    if (response.status !== STATUS_CODE.OK) {
+      console.error(jsonReponse.message);
+      throw jsonReponse.message;
+    }
+
+    return jsonReponse.isActive;
+  }
+
+  async getBestuursperiode(mandatarisId) {
+    const response = await fetch(
+      `${API.MANDATARIS_SERVICE}/mandatarissen/${mandatarisId}/bestuursperiode`
+    );
+    const jsonReponse = await response.json();
+
+    if (response.status !== STATUS_CODE.OK) {
+      console.error(jsonReponse.message);
+      throw jsonReponse.message;
+    }
+
+    return {
+      uri: jsonReponse.uri,
+      id: jsonReponse.id,
```

Comment:
As in this the usecase returning the model would make the mandataris service easier?